app/models/spotlight/reindexing_log_entry.rb

```diff
@@ -0,0 +1,42 @@
+module Spotlight
+  ##
+  # a log entry representing an attempt to reindex some number of records in an exhibit
+  class ReindexingLogEntry < ActiveRecord::Base
+    enum job_status: { unstarted: 0, in_progress: 1, succeeded: 2, failed: 3 }
+
+    belongs_to :exhibit, class_name: 'Spotlight::Exhibit'
+    belongs_to :user, class_name: '::User'
+
+    # null start times sort to the top, to more easily surface pending reindexing
+    default_scope { order('start_time IS NOT NULL, start_time DESC') }
+    scope :recent, -> { limit(5) }
+
+    def duration
+      end_time - start_time if end_time
+    end
+
+    def reindexing_started!
+      self.start_time = Time.zone.now
+      self.job_status = :in_progress
+      save
+    rescue
+      Rails.logger.error "unexpected error updating log entry to :in_progress from #{caller}"
+    end
+
+    def reindexing_succeeded!
+      self.end_time = Time.zone.now
+      self.job_status = :succeeded
```

Comment:
Yeah, I was wondering if @jmartin-sul was trying to be more explicit about the fact that `end_time` and `job_status` would be changing when `reindexing_succeeded!` was called.  Otherwise the save is a side affect of the `succeeded!` call and not immediately obvious when looking at this code (unless you know the particulars of `ActiveRecord#enum`.

I'm on the fence either way.